packages/web/src/components/premium-content-purchase-modal/components/PurchaseContentFormFields.tsx

```diff
@@ -95,30 +99,38 @@ export const PurchaseContentFormFields = ({
   }
 
   const options = [
-    {
-      label: messages.existingBalance,
-      id: PurchaseMethod.EXISTING_BALANCE,
-      icon: IconDonate,
-      disabled: isExistingBalanceDisabled,
-      value: (
-        <Text
-          variant='title'
-          color={
-            purchaseMethod === PurchaseMethod.EXISTING_BALANCE
-              ? 'secondary'
-              : undefined
-          }
-        >
-          {`$${USDC(balanceUSDC).toFixed(2)}`}
-        </Text>
-      )
-    },
+    hasBalance
```

Comment:
i think we actually want to always display this row, but have it disabled if the existing balance is not enough to cover the transaction.

Reply:
yes but this is the case where you have no balance so you wouldn't even understand what this was! if you have $1 and it's not enough, we do the gray out thing. this is consistent w how it was done in mobile already!